### PR TITLE
Retrofit buildings

### PR DESCRIPTION
**Pull request by Phaedra144**
Please check the retrofit extension, and please merge if fine.

Thanks, Szilvi

### CODE REVIEW

**Comment by adamgyulavari** [comment]
@Phaedra144 resolve conflicts

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/model/api/ApiService.java [hunk 1]
```diff
@@ -24,4 +28,19 @@
     @GET("/kingdom/{userId}/")
     Call<Kingdom> getKingdom(@Path("userId") int userId);
 
+    @GET("/kingdom/{userId}/buildings/")
+    Call<ArrayList<Building>> getBuildings(@Path("userId") int userId);
```
Comment (adamgyulavari) [inline app/src/main/java/com/greenfox/peridot/peridot_coz_android/model/api/ApiService.java]: @Phaedra144 this is not what the API will give you back

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/greenfox/peridot/peridot_coz_android/model/api/ApiService.java (hunk 1 above)
Comment (radaladar) [reply app/src/main/java/com/greenfox/peridot/peridot_coz_android/model/api/ApiService.java]: @adamgyulavari BuildingsResponse created on listview-buildings branch

**Comment by radaladar** [comment]
pls resolve conflicts

**Comment by Phaedra144** [comment]
Done ;)

**Pull request merged**